### ISSUE TITLE
Front-End: Link not found on page/Accessibility New Window Warnings

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#188]
I'm not sure we can do anything about this, but curious for thoughts - will the switch to JS checking fix this?

This link cannot be found because they have Accessibility New Window Warnings installed and it's adding the warning on page load.

<img width="1016" alt="Screen Shot 2023-06-23 at 9 18 32 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/954a837b-6b5c-4077-b4e6-886bac18dbdc">

**Comment by boonedev** [comment]
@amberhinds I think it should work fine once we switch to the JS check.

**Comment by SteveJonesDev** [comment]
Issue moved to our internal system.
